src/config/index.ts

```diff
@@ -81,6 +81,6 @@ export default {
   parserVersion: process.env.PARSER_VERSION || 'v3beta',
   pagination: {
     defaultPageSize: 30,
-    maxPageSize: 100
-  }
+    maxPageSize: 100,
+  },
```

Comment:
swooping in to note that updating the linter to our [latest standards](https://github.com/Pocket/collection-api/pull/618) may solve this.